src/components/Button/src/Button.vue

```diff
@@ -681,7 +681,7 @@ export default {
 }
 
 .InformationText {
-	width: min-content;
+	flex: 0 0 fit-content;
```

Comment:
Any reason for using `fit-contnent` over `auto`. Auto is the default and I think slightly better supported.